kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -218,8 +218,9 @@
     max-width: 200px;
     // overflow-x hidden seems to affect overflow-y also, so include a fixed height
     height: 16px;
-    overflow-x: hidden;
-    overflow-y: hidden;
+    // overflow: hidden on both x and y so that the -y doesn't show scroll buttons
+    // at certain zooms/screen sizes
+    overflow: hidden;
```

Comment:
Maybe to be safe, keep the two-line version of this rule. It's very possible that

```
overflow-x: hidden;
overflow-y: hidden;
```

is the best way to avoid the various cross-browser issues we've had with the username. I can give this a quick test on firefox though.

Reply:
Actually looks fine. I wonder why we changed it from `overflow` to just `overflow-x`.